noodles/run/logging.py

```diff
@@ -3,12 +3,34 @@
 of streams.
 """
 
+import sys
 import logging
 
 from ..lib import (EndOfQueue)
 from ..workflow import (is_workflow)
 from .messages import (JobMessage, ResultMessage)
 
+__all__ = ["make_logger", "logger"]
+
+
+def _initialize_logger() -> logging.Logger:
+    """Initialize the ``noodles`` logger and set its stream handlers."""
+    logger = logging.getLogger("noodles")
+    logger.setLevel(logging.INFO)
+
+    stream_handler = logging.StreamHandler()
+    stream_handler.setLevel(logging.INFO)
+    stream_handler.setFormatter(logging.Formatter(
+        fmt="[%(asctime)s] %(levelname)s: %(message)s",
+        datefmt="%H:%M:%S",
+    ))
```

Comment:
By default `logging.StreamHandler` will output to `stderr`. Other streams (`stdout`) are also a possiblity, or something like picking a stream based on the level, but for the sake of simplicity I personally have a slight preference for `stderr`.

Any thoughts?